lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -242,6 +253,16 @@ extension DomCSSStyleDeclarationExtension on DomCSSStyleDeclaration {
   set backgroundImage(String value) =>
       setProperty('background-image', value, '');
   set border(String value) => setProperty('border', value, '');
+  set mixBlendMode(String value) => setProperty('mix-blend-mode', value, '');
```

Comment:
I don't think we need the last arg here either, convenience method takes care of it